.github/workflows/main.yml

```diff
@@ -0,0 +1,134 @@
+name: CI Pipeline
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'python-flask/**'
+      - 'c-sharp-razor/**'
+      - 'rust-actix/**'
+      - 'ruby-sinatra/**'
+      - 'javascript-express/**'
+      - 'go-gin/**'
+      - 'go-gorilla/**'
+      - '/**'
```

Comment:
I'm not totally sure why it runs all the tests when something in the root changes. Maybe we should only run all the tests when something in the /database and the /.env.local is changed? But honestly no string opinion on that.

Reply:
My thinking was this:

root: makefile and env vars -> influences all -> should run all tests

but actually you are right that all the tests should run if the /data-folder changes so I will add that :)

Reply:
works as intended